### DISCUSSION TITLE
Is i18n still in the pipeline?

### DISCUSSION THREAD

**Discussion by krillboi** [facebook/docusaurus#3000] [General] [answered]
We are looking to add support for different languages on support.touchgfx.com in the not so distant future, so I would just like to hear what your current plans for implementing this feature is? 

I don't think I've said this enough, but working with Docusaurus has so far been a joy and you've really created something great here. So cheers to you all! :)

**Comment by slorber** [comment, marked as answer]
100% sure, I'll start working on this in July

**Reply by krillboi** [reply]
Sounds good!

**Reply by krillboi** [reply]
@slorber do you have any idea what the solution will look like or is it still in planning? Just to get an idea of how the workflow will be :)

**Reply by slorber** [reply]
Need to do some tests but I'd like it to be not too far from the workflow of v1. markdown files could be uploaded directly to translation saas like crowdin, and pages translations would be extracted from code (we might use FBT for that, instead of the `<translate>` tags in v1).

We need to test a bit of things first (like seeing how saas tools support uploading MDX etc). Not very familiar with v1 translation system or Crowdin yet...

**Comment by krillboi** [comment]
@slorber sorry to keep asking you, but is there a current estimate on when this will be available? Maybe just a rough one.

**Reply by slorber** [reply]
@krillboi in a month you could have a preview, and 2 months something more stable

Can't be sure, I still have work to do on the versioning features first

**Reply by krillboi** [reply]
That's fine, thanks for the reply!
